src/main/java/com/icthh/xm/ms/dashboard/repository/DashboardRepository.java

```diff
@@ -23,4 +27,8 @@ public interface DashboardRepository extends ResourceRepository {
     List<Dashboard> findAllById(Set<Long> ids);
 
     <S extends Dashboard> List<S> saveAll(Iterable<S> entities);
+
+    Page<Map<String, Object>> findAllAudits(Pageable pageable);
+
+    Page<Map<String, Object>> findAuditsById(Long id, Pageable pageable);
```

Comment:
Why we return Object here?

Reply:
because the Dashboard is not returned, but the object {"revInfo": RevInfo.class, "audit": Dashboard.class, "operation": RevisionType.class}